src/chrome/chromeConnection.ts

```diff
@@ -31,6 +31,8 @@ export interface ITarget {
 
 export type ITargetFilter = (target: ITarget) => boolean;
 export interface ITargetDiscoveryStrategy {
+    version: Promise<ProtocolSchema>;
```

Comment:
I don't think `version` belongs as a property on this. It's otherwise stateless, ie you call `getTarget`/`getAllTargets` and get info back. This property would be "the version of the last endpoint I accessed" which doesn't feel right. So I think it would be better for `getTarget/getAllTargets` to return a result object that contains the ITargets and also the version. Does that make sense?